pkg/config/templates/rancherd-10-harvester.yaml

```diff
@@ -48,6 +48,9 @@ bootstrapResources:
   metadata:
     name: rancher-expose
     namespace: cattle-system
+    annotations:
+      nginx.ingress.kubernetes.io/proxy-body-size: "0"
+      nginx.ingress.kubernetes.io/proxy-request-buffering: "off"
```

Comment:
prefer to set those values via the `HelmChartConfig` using https://github.com/kubernetes/ingress-nginx/blob/9c0a39636da11b7e262ddf0b4548c79ae9fa1667/charts/ingress-nginx/values.yaml#L27, e.g,

```YAML
  spec:
    valuesContent: |-
      controller:
        config:
          proxy-body-size: "0"
          proxy-request-buffering: "off"
```